editor/state.js

```diff
@@ -303,6 +303,21 @@ export function isSidebarOpened( state = false, action ) {
 	return state;
 }
 
+export function editorWidth( state = null, action ) {
+	const layout = document.querySelector( '.editor-layout__content' );
```

Comment:
>which doesn't seem great.

In what way? We're not really computing much. We could persist references to the DOM nodes. Apparently `offsetWidth` [can force a reflow](https://gist.github.com/paulirish/5d52fb081b3570c81e3a#box-metrics), but if it's tied to resize event, shouldn't be a concern (or, rather, we're going to have to call it anyways).